src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -591,6 +597,17 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         return api.getUsersAPI(false).getAvatarUrlWithAccessToken(userId, file.id);
     };
 
+    handleAnnotationSelect = (annotation: Annotation): void => {
+        const {
+            file_version: { id: fileVersionId },
+        } = annotation;
+        const { onAnnotationSelect } = this.props;
+        const { feedItems } = this.state;
+        const version = feedItems.filter(item => item.type === 'file_version').find(item => item.id === fileVersionId);
```

Comment:
Is it possible for the version to not yet be loaded? If we introduce pagination to the feed calls, will this continue to work? What happens if the version isn't present?

Reply:
Currently it's not possible for the version to not yet be loaded -- except for the case where there is only 1 version of the file. Once we introduce pagination, we may need to dynamically fetch the missing version. That or if we are able to refactor the way ContentPreview loads previous versions, we may be able to get away with just sending the version id instead of the whole object. If the version isn't present, the logic upstream just treats it as a change in selected annotation id on the current version